src/software/thunderscope/common/proto_plotter.py

```diff
@@ -1,26 +1,25 @@
 from random import randint
 import time
 from collections import deque
+from typing import Type, Callable
```

Comment:
Do we still need to import `Callable` and `Type` from `typing` with `python3.10`

Reply:
Yes